src/bots/depositor.py

```diff
@@ -25,15 +25,16 @@
 from transport.msg_types.deposit import DepositMessage, DepositMessageSchema, get_deposit_messages_sign_filter
 from transport.msg_types.ping import PingMessageSchema, to_check_sum_address
 from transport.types import TransportType
-from web3.contract.contract import ContractFunction
 from web3.types import BlockData
 
 logger = logging.getLogger(__name__)
 
 
 def run_depositor(w3):
     logger.info({'msg': 'Initialize Depositor bot.'})
-    depositor_bot = DepositorBot(w3)
+    gas_price_calculator = GasPriceCalculator(w3)
```

Comment:
Some services initialized outised __init__ and some inside. Let's initialize all classes in one method

Reply:
Done